integrations/docker/images/chip-cert-bins/Dockerfile

```diff
@@ -0,0 +1,198 @@
+    && cmake ../ \
+    && make -j8 \
+    && make install \
+    && cd ../.. \
+    && rm -rf bloaty \
+    && : # last line
+
+# NodeJS: install a newer version than what apt-get would read
+# This installs the latest LTS version of nodejs
+RUN case ${TARGETPLATFORM} in \
+        "linux/amd64") \
+            set -x \
+			&& mkdir node_js \
+			&& cd node_js \
+			&& wget https://nodejs.org/dist/v12.19.0/node-v12.19.0-linux-x64.tar.xz \
+			&& tar xfvJ node-v12.19.0-linux-x64.tar.xz \
+			&& mv node-v12.19.0-linux-x64 /opt/ \
+			&& ln -s /opt/node-v12.19.0-linux-x64 /opt/node \
+			&& ln -s /opt/node/bin/* /usr/bin \
+			&& cd .. \
+			&& rm -rf node_js \
+			;; \
+        "linux/arm64")\
+            set -x \
+			&& mkdir node_js \
+			&& cd node_js \
+			&& wget https://nodejs.org/dist/v12.19.0/node-v12.19.0-linux-arm64.tar.xz \
+			&& tar xfvJ node-v12.19.0-linux-arm64.tar.xz \
+			&& mv node-v12.19.0-linux-arm64 /opt/ \
+			&& ln -s /opt/node-v12.19.0-linux-arm64 /opt/node \
+			&& ln -s /opt/node/bin/* /usr/bin \
+			&& cd .. \
+			&& rm -rf node_js \
+			;; \
+        *) ;; \
+    esac
+
+# Set python to python3
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10
+
+RUN mkdir /root/connectedhomeip
+ADD ./ /root/connectedhomeip/
+WORKDIR /root/connectedhomeip/
+RUN scripts/build/gn_bootstrap.sh
+RUN gn gen out/debug --args='chip_mdns="platform" chip_inet_config_enable_ipv4=false'
+RUN ninja -C out/debug
+
+RUN scripts/examples/gn_build_example.sh examples/all-clusters-app/linux/ examples/all-clusters-app/linux/out/all-clusters-app chip_inet_config_enable_ipv4=false
+RUN scripts/examples/gn_build_example.sh examples/bridge-app/linux/ examples/bridge-app/linux/out/bridge-app chip_inet_config_enable_ipv4=false
+RUN scripts/examples/gn_build_example.sh examples/tv-app/linux/ examples/tv-app/linux/out/tv-app chip_inet_config_enable_ipv4=false
+RUN scripts/examples/gn_build_example.sh examples/tv-casting-app/linux/ examples/tv-casting-app/linux/out/tv-casting-app chip_inet_config_enable_ipv4=false
+RUN scripts/examples/gn_build_example.sh examples/lighting-app/linux/ examples/lighting-app/linux/out/lighting-app chip_inet_config_enable_ipv4=false
+RUN scripts/examples/gn_build_example.sh examples/thermostat/linux/ examples/thermostat/linux/out/thermostat chip_inet_config_enable_ipv4=false
+RUN scripts/examples/gn_build_example.sh examples/ota-provider-app/linux examples/ota-provider-app/linux/out/host 'chip_config_network_layer_ble=false'
+RUN scripts/examples/gn_build_example.sh examples/ota-requestor-app/linux examples/ota-requestor-app/linux/out/host 'chip_config_network_layer_ble=false'
+RUN scripts/examples/gn_build_example.sh examples/door-lock-app/linux/ out/door-lock-app chip_inet_config_enable_ipv4=false
+
+RUN npm --prefix third_party/zap/repo/ ci
+RUN scripts/examples/gn_build_test_example.sh app1
+
+# Stage 2: Copy relevant test harness bins to a new image
+FROM ubuntu:21.10
+ENV TZ=America/Los_Angeles
```

Comment:
why is the timezone update needed and how did we pick it?

Reply:
One of the packages requires it, I believe it is libssl? This is what's used when building the test harness, so we are using the same America/Los_Angeles TZ here.